main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"net"
+	"os"
+
+	pb "github.com/kubearmor/KVMService/protobuf"
+	"google.golang.org/grpc"
+)
+
+type server struct {
+	pb.UnimplementedKVMServer
+}
+
+func (s *server) RegisterAgentIdentity(ctx context.Context, in *pb.AgentIdentity) (*pb.Status, error) {
+	log.Println("Agent Identity is ", in.Identity)
```

Comment:
@nam-jaehyun , is it ok to use `log` for logging?

Reply:
This is temporary Rahul. Just for development. WIll change later